docs/en/cloud/Alibaba-Cloud-ACK.md

```diff
@@ -106,40 +106,49 @@ After the download is complete, unzip the package and set env "`SPARK_HOME`":
+This image needs to include the Alluxio client jar package. 
+You can obtain the Alluxio client jar package as follows:
 ```console
-$ id=$(docker create alluxio/alluxio-enterprise:2.2.1-1.4)
-$ docker cp $id:/opt/alluxio/client/alluxio-enterprise-2.2.1-1.4-client.jar \
-	$SPARK_HOME/jars/alluxio-enterprise-2.2.1-1.4-client.jar
+$ id=$(docker create alluxio/alluxio:{{site.ALLUXIO_VERSION_STRING}})
+$ docker cp $id:/opt/alluxio/client/alluxio-{{site.ALLUXIO_VERSION_STRING}}-client.jar \
+	$SPARK_HOME/jars/alluxio-{{site.ALLUXIO_VERSION_STRING}}-client.jar
 $ docker rm -v $id 1>/dev/null
 ```
-After the alluxio client jar package is ready, start building the image:
+
+After the Alluxio client jar package is ready, start building the image:
 ```console
 $ docker build -t \
   spark-alluxio:2.4.6 -f $SPARK_HOME/kubernetes/dockerfiles/spark/Dockerfile $SPARK_HOME
 ```
-After the image is built, there are two ways to process the image:
+You can find more details about running Alluxio with Spark on Kubernetes [here]({{ '/en/compute/Spark-On-Kubernetes.html' | relativize_url }}).
+
+After the image is built, there are two ways to distribute the image:
 
-* If there is a private image warehouse, push the image to the private image warehouse, and ensure that the k8s cluster node can pull the image.
-* If there is no private image warehouse, you need to use the docker save command to export the image, and then scp to each node of the k8s cluster, use the docker load command on each node to import the image, so that you can ensure that each node exists The mirror.
+* If there is a private image warehouse, push the image to the private image warehouse, and ensure that the K8s cluster node can pull the image.
+* If there is no private image warehouse, you need to use the `docker save` command to export the image, then scp the image to each node of the K8s cluster, use the `docker load` command on each node to load the image.
  {% endcollapsible %}
 
  {% collapsible Upload files to Alluxio %}
-As mentioned at the beginning of the document: This experiment is to submit a spark job to k8s and the goal of the spark job is to count the number of occurrences of each word for a certain file. Now you need to upload the file to the alluxio storage. Here, for convenience, you can directly upload the file `/opt/alluxio-{{site.ALLUXIO_VERSION_STRING}}/LICENSE` in the alluxio master (the file path may be slightly different due to the alluxio version) to alluxio.
+As mentioned at the beginning: In this experiment we will submit a Spark job to K8s. 
+The Spark job will perform a word count calculation on a certain file. 
+Before we kick off the Spark job, we need to upload the file to the Alluxio storage. 
+Here, for convenience, we directly upload the file `/opt/alluxio-{{site.ALLUXIO_VERSION_STRING}}/LICENSE` from the Alluxio master node (the file path may be slightly different due to the Alluxio version) to Alluxio.
```

Comment:
from alluxio master node to alluxio?

Reply:
What about `from the Alluxio master node to the Alluxio namespace`? It means `copyFromLocal` from the node into Alluxio.